ports/mimxrt10xx/common-hal/audiobusio/I2SOut.c

```diff
@@ -121,6 +130,11 @@ void common_hal_audiobusio_i2sout_deinit(audiobusio_i2sout_obj_t *self) {
 
     common_hal_reset_pin(self->data);
     self->data = NULL;
+
+    IOMUXC_GPR->GPR1 &= ~(IOMUXC_GPR_GPR1_SAI1_MCLK_DIR_MASK << (self->instance - 1));
+
+    common_hal_reset_pin(self->mclk);
```

Comment:
does this need to be conditional on `if(self->mclk)`? Otherwise it might affect a bit in GPR1 that it sholdn't